modules/update-upgrading-cli.adoc

```diff
@@ -54,6 +53,12 @@ Recommended updates:
 For details and information on how to perform an `EUS-to-EUS` channel update, please refer to the
 _Preparing to perform an EUS-to-EUS upgrade_ page, listed in the Additional resources section.
 ====
++
+[NOTE]
+====
+If there are no available updates, updates that are supported but not recommended might still be available.
+See _Updating along a conditional update path_ for more information.
+====
```

Comment:
Consider either moving one of these notes or else combining them (like, one `NOTE` block with two bullets or something) to avoid stacked admonitions

Reply:
I don't think there are many steps left in this procedure that don't already have their own admonition(s) 😅 so I'm happy to have a single admonition block here with two bullet points